lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -61,7 +90,15 @@ public static String convertGpaToLetterGrade(double gpa) {
    * @return The factorial of n.
    */
   public static int computeFactorial(int n) {
```

Comment:
Looks good! I'd be curious to know if you can implement this recursively. Give it a shot.

Reply:
Got this working recursively....definitely was a little mind bending but it's pretty cool how it works.

Reply:
Yessir! And excellent work with this.

nit: Please resolve the conversation when you're done with it. Once all conversations are resolved, you should be able to send back to me and I'll see your PR as ready to review.